### ISSUE TITLE
Incorrect password after latest update

### ISSUE THREAD

**Issue by skcuslleb** [dustindclark/homebridge-envisalink#7]
After the last update of index.js I was getting the Incorrect Password error. After digging into for a couple of days I finally broke out the packet sniffer and realized the problem was introduced with the following 2 lines

`nap.manualCommand("000");`
`nap.manualCommand("001");`

I commented those out and everything seems to work again.  What exactly are the purpose of those lines?  I can't quite figure out why they were added.

**Comment by winescb** [comment]
Same boat here. What file are those commands located in. I did try searching but pretty new to atom and js, not very proficient yet.

Thanks

**Comment by skcuslleb** [comment]
> Same boat here. What file are those commands located in. I did try searching but pretty new to atom and js, not very proficient yet.
> 
> Thanks

index.js

**Comment by trilidar** [comment]
Completely lost.

I am unable to locate index.js for anything (once installed).

Could someone please provide a dir list of where this stuff lives?

**Comment by trilidar** [comment]
Think I may have found it, but this line appears to be the problem (nodeproxy's index.js):

```
    if (data.substring(3,4) == '3') {
        console.log('login requested... sending response...');
        sendcommand(actual,'005'+config.password);
```

My error show:

server bound
actual connected
Login Request Response: 3 
login requested... sending response...
sendcommand 005mypasswordA4
Command 005 Acknowledged
Login Request Response: 0 
Incorrect Password :(
actual disconnected

If I eliminate the 005 from the index.js, I'm still met with a mypassword0F (note the change from A4).  I've been unable to locate where/how that's being added.

**Comment by skcuslleb** [comment]
That line is definitely not the problem. You need to find index.js in homebridge-envisalink. For me it lives here

/usr/local/lib/node_modules/homebridge-envisalink

**Comment by lsmith07** [comment]
I had the same issue (but didn't realize it for 1.5 days).  I have an Evisalink 4 connected to PC1616, with two RFK5501 keypads, and homebridge running on rpi3 with two other plugins.  I could control my alarm using my EYEZON account and by connecting to EVL4 directly (using a browser and its IP address); however, I couldn't get this plugin to work.  It caused homebridge to stop responding to HomeKit, and my other plugins were unresponsive as a result..until many hours later when I read this post and commented out those two lines.  Doing so fixed it completely; everything works correctly now.  I wish I had read this post yesterday morning.  Thank you afrusagli!!!

dustindclark, first of all, Thank you for publishing this plugin!  It is really awesome once getting it to work, but please publish the index.js file without those two lines so future poor souls won't waste so many hours like me.

**Comment by dustindclark** [comment]
Fixed and published in 0.1.4.  Please give it a shot and let me know.  Thanks.
